docs/topics_5/web-adding-global-custom-rules.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_rules_{context}']
+= Adding global custom rules
+
+Global rules apply to all projects and are automatically applied to analyses for which they fit the definitions of a project.
+
+The {ProductShortName} includes a preconfigured set of global rules and you can define your own custom global rules.
+
+These rules are displayed and defined on the *Rules configuration* page, which contains two tabs: *System rules* and *Custom rules*. *System rules* contains a preconfigured set of global rules. You can add your own rules in the *Custom rules* tab.
+
+For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
+
+.Rules configuration
+image::web-rules-config_51.png[Rules configuation]
+
+Each row in the tables displayed in *Rules configuration* shows the short path, source, target, and the number of rules in the rule set.
+
+You can upload rule sets or register a server path that contains a rule set.
+
+
+.Procedure
+In the navigation pane, click *Rules configuration*.
+
+. Click *Add rules*.
+. To upload rules:
```

Comment:
I think you could keep a single line like you did in the last version

If you want to upload a rules file, click Browse, ... and the click Close.
+
Explanation about why you want to use a rules file. You were going to add it here.